suc/pkg/service/common.go

```diff
@@ -0,0 +1,51 @@
+package service
+
+import (
+	"golang.org/x/sys/windows/svc"
+)
+
+func SlicesMatch(s1 []string, s2 []string) bool {
```

Comment:
This function is only used once, is it written this way because the ordering doesn't matter? I feel like if the ordering is always the same and does matter, it would be easier to use `reflect.DeepEqual` or compare each index of the slice. 

In other words, if the time complexity of this function is on purpose, it should have a comment and test cases and maybe update the name to `UnorderedSlicesEqual` or something similar (honestly I would use generics for this but that's just me), and it it's not, you probably don't need the function at all and could use `reflect.DeepEqual` wherever this function is called.

Reply:
I couldn't find documentation that guaranteed the windows api would return dependencies in the same order each time, so I'd rather leave this as unordered. I've updated this to use generics and I've added a unit test case